projects/packages/videopress/src/client/components/timestamp-control/style.module.scss

```diff
@@ -1,8 +1,13 @@
 .timestamp-control__controls-wrapper {
+	--input-width: 24px;
 	display: flex;
 	align-items: center;
 	gap: 8px;
-	margin-bottom: 8px;
+	flex-wrap: wrap;
+
+	.timestamp-range-control {
+		min-width: 270px;
+	}
```

Comment:
It doesn't seem to be needed anymore.

```suggestion
```

Reply:
That is the input range. WIthout this, the `flex-wrap` is not triggered:
![2023-03-23_16-21-23](https://user-images.githubusercontent.com/8486249/227325813-a8842066-d099-46a1-84ff-a07f10960202.png)

Reply:
The intended result is this, with more room for the slider:
![2023-03-23_16-25-49](https://user-images.githubusercontent.com/8486249/227326952-f8755ba4-a4ae-4d4a-a34d-3f40a4177d2a.png)

Let me know if you think leaving it at the side is better.

Reply:
Considering the block sidebar has 250px approx we can assume that the slider will be put down. I think we should try to keep it next to the inputs.